scripts/github_runner/setup_bare_metal_runner.sh

```diff
@@ -22,19 +22,25 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-if [ -z $REPO_NAME ] || [ -z $RUNNER_TOKEN ] || [ -z $RUNNER_LABEL ] || [ -z $RUNNER_NAME ] || [ -z $SANDBOX ]; then
-    echo "Parameters missing"
-    exit -1
+set -Eeuo pipefail
+
+cd "$( dirname "${BASH_SOURCE[0]}" )"
+
+if (( $# != 5)); then
+    echo "Invalid number of parameters"
+    echo "USAGE: start_bare-metal_runner.sh <vHive repository account> <runner token> <runner label> <runner name> <sandbox>"
+    exit 1
 fi
 
-URL="https://github.com/${REPO_NAME}"
+REPO_ACCOUNT=$1
+RUNNER_TOKEN=$2
+RUNNER_LABEL=$3
+RUNNER_NAME=$4
+SANDBOX=$5
 
-sudo apt-get update
+ROOT="./../.."
```

Comment:
I feel relative paths can be confusing, can we have a absolute path here?

Reply:
IMO, it's better as is.